versioned_docs/version-8.3/apis-tools/operate-api/overview.md

```diff
@@ -487,6 +487,7 @@ These values could be of type `string`, `number`, `boolean`, and `dateString`.
  "state":                   <string>
 }
 ```
+HERE I WOULD HAVE EXPECTE AN EXPLANATION WHAT THE DIFFERENT STATE MEAN. ESPECIALLY 'PENDING' IS NOT DEFINED IN THE DOCS.
```

Comment:
@falko There is actually another [PR](https://github.com/camunda/operate/issues/4648), I'm also looking into, well started, but then 8.3 focus happened so it stalled. I plan on picking that up soon again and since it closely touches same area I will look at both together. 
The quickest suggestion here would be that I perhaps just add some extra details to docs, as well as swagger to explain state enum (and probably others as well while I'm at it). I can add you as reviewer when at that point if you like?